src/server/ssrServer.js

```diff
@@ -96,13 +96,15 @@ export function createApp({ enablePostToModuleRoutes = false } = {}) {
       '*',
       addSecurityHeaders,
       json({ limit: '0kb' }), // there should be no body
+      urlencoded({ limit: '0kb' }), // there should be no body
       cors({ origin: false }) // disable CORS
     );
 
     app.post(
       '*',
       addSecurityHeaders,
       json({ limit: '15kb' }),
+      urlencoded({ limit: '15kb' }),
```

Comment:
If we are okay with the limit and feel secure about supporting this, I think this would be a good feature to bring in!
@sumanramagoni can you please add documentation and I think this is a feature, not a fix btw.

Reply:
yea, I've realized at later point in time that this should be considered as feature. 
shall I close this pr and raise new one with feature?

Reply:
you can edit the pull request and update the commit message, no need to close the PR

Reply:
done.
I added documentation as well.